ViewModel/EnvironmentAndExtendStoreUuid.php

```diff
@@ -34,10 +34,7 @@ public function __construct(
     public function getActiveEnvironment()
     {
         $activeEnvironmentUrl = $this->activeEnvironmentURLBuilder->getIntegrationURL();
```

Comment:
Should be able to resolve this conflict by reverting and merging in master

Reply:
fixed